### PR TITLE
Update profile page

### LINKED ISSUE #154: Zmiany z formularzu zmiany hasła
Do poprawy z #153:
- [ ] sygnalizacja powodzenia/niepowodzenia (akcja jest wysyłana, nie jest wyświetlana),
- [ ] wysyłanie formularza klawiszem,
- [x] zablokowanie strony profilu dla niezalogowanych.
- [ ] po wejściu na stronę pole old password jest zaznaczone jako błędne

### PR DESCRIPTION
**Pull request by kofeinaa**
Closes #154.

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -196,6 +196,7 @@ export default function createRoutes(store) {
     }, {
       path: profilePage(),
       name: 'Profile',
+      onEnter: requireAuth,
```
Comment (maxmati) [inline app/routes.js:199]: This looks redundant with for me with:
https://github.com/zmora-agh/zmora-ui/pull/233/files#diff-71bee332c774afdddad7edcfe3762201R21

File: app/containers/ProfilePage/index.js [hunk 2]
```diff
@@ -6,28 +6,40 @@
...
 import PasswordChangeForm from '../../components/PasswordChangeForm';
 import { changePassword } from './actions';
+import { passwordChangePropType } from '../../components/PasswordChangeForm/constants';
+
 
 function ProfilePage(props) {
   return (
     <div>
       <PasswordChangeForm
         onSubmit={(oldPassword, password) => props.dispatch(changePassword(oldPassword, password))}
```
Comment (maxmati) [inline app/containers/ProfilePage/index.js:22]: Did you considered changing saga to `@graphql` like we do in other places where we can? Then exporting props to const, whole machinery for storing and fetching data from store shouldn't be necessary.

File: app/components/PasswordChangeForm/index.js [hunk 3]
```diff
@@ -1,29 +1,36 @@
...
 /* import { LinearProgress } from 'material-ui/Progress'; */
 import ErrorTextField from '../ErrorTextField';
 import messages from './messages';
 import { PASSWORD_PATTERN } from '../RegisterForm/constants';
 import { componentRequireAuth } from '../../utils/auth';
+import { passwordChangePropType } from './constants';
+
 
 @componentRequireAuth
 class PasswordChangeForm extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
```
Comment (maxmati) [inline app/components/PasswordChangeForm/index.js:21]: Can you add `LinearProgress` to be consistent with Register and Login forms?

Comment (kofeinaa) [reply app/components/PasswordChangeForm/index.js:21]: I can, but, what's the purpose?

Comment (maxmati) [reply app/components/PasswordChangeForm/index.js:21]: To be visualy  consistent with other forms and to give user any feedback that something is happening in the background.

**Pull request merged**